CHANGELOG.md

```diff
@@ -6,7 +6,7 @@
 
 ### Fixed
 * <How do the end-user experience this issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
-* None.
+* The application could crash when removing backlinks. We will no longer assert that the backlink is found, and thus not cause a crash. ([#6585](https://github.com/realm/realm-core/issues/6585), v6.0.0)
```

Comment:
I'd suggest being more specific, like: we could crash when removing backlinks in cases where forward links did not have a corresponding backlink due to corruption. We now silently ignore this inconsistency allowing the app to continue" or similar.